e2e/libs/keywords/recurringjob_keywords.py

```diff
@@ -33,6 +33,13 @@ def create_backup_recurringjob_for_volume(self, volume_name):
         self.recurringjob.create(job_name, task="backup")
         self.recurringjob.add_to_volume(job_name, volume_name)
 
+    def create_recurringjob_for_volume(self, volume_name, task, cron="*/2 * * * *"):
```

Comment:
Can these 3 similar functions: `create_snapshot_recurringjob_for_volume`, `create_backup_recurringjob_for_volume` and `create_recurringjob_for_volume` be merged into one single function?